modules/ROOT/pages/dw-core-functions-endswith.adoc

```diff
@@ -3,31 +3,45 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<endswith1>>
-
 
 [[endswith1]]
-== endsWith(String, String): String
+== endsWith(String, String): Boolean
+
+Returns `true` if a string ends with a provided substring, `false` if not.
+
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `text` | The input string (a `String`).
+| `suffix` | The suffix string to find at the end of the input string.
+|===
 
-Returns true or false depending on if a string ends with a provided substring.
+=== Example
 
-.Transform
-[source,dataweave,linenums]
+This example finds "no" (but not "to") at the end of "Mariano".
+
+==== Source
+
+[source,DataWeave, linenums]
 ----
 %dw 2.0
 output application/json
 ---
 {
-  a: "Mariano" endsWith "no",
-  b: "Mariano" endsWith "to"
+  "yes" : "Mariano" endsWith "no",
+  "no" : "Mariano" endsWith "to"
```

Comment:
I find the names "yes" and "no" confusing, especially since the string you are looking for is also "no."